.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "yarn" # See documentation for possible values
```

Think this should be
```suggestion
  - package-ecosystem: "npm" # See documentation for possible values
```

---

ahh you're right, misread the docs 🥂